### PR TITLE
Enabling Fee Reciept and File Pay button for minimal changes in the f…

### PR DESCRIPTION
**Pull request by Rajandeep98**
…iling

*Issue #24945:* /bcgov/entity#24945

*Description of changes:*

- enable file and pay button for details
- enable fee summary in all cases

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/common/Actions.vue [hunk 1]
```diff
@@ -76,7 +76,7 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Name
   @Getter(useStore) getFilingId!: number
   // @Getter(useStore) getNameRequestNumber!: string
   @Getter(useStore) hasAlterationDataChanged!: boolean // for testing state-getters
-  @Getter(useStore) hasCorrectionDataChanged!: boolean
+  // @Getter(useStore) hasCorrectionDataChanged!: boolean
```
Comment (severinbeauvais) [inline src/components/common/Actions.vue:79]: Delete this altogether if it's no longer used in here.

File: src/store/store.ts [hunk 2]
```diff
@@ -565,44 +565,44 @@ export const useStore = defineStore('store', {
...
-          this.haveOfficeAddressesChanged ||
-          this.havePeopleAndRolesChanged ||
-          this.hasSpecialResolutionRulesChanged ||
-          this.hasSpecialResolutionResolutionChanged
-        )
-      }
-
-      return false // should never happen
-    },
+    // hasCorrectionDataChanged (): boolean {
```
Comment (severinbeauvais) [inline src/store/store.ts:568]: Add a comment that this is no longer used but may be useful in the future.

File: src/store/store.ts [hunk 3]
```diff
@@ -1205,7 +1205,7 @@ export const useStore = defineStore('store', {
         waiveFees: false
       }]
       const haveFilingChange = (
-        (this.isCorrectionFiling && this.hasCorrectionDataChanged) ||
+        (this.isCorrectionFiling) ||
```
Comment (severinbeauvais) [inline src/store/store.ts:1208]: Add a comment that we always want to show the Fee Summary if this is a correction filing.

File: tests/unit/CoopCorrection.spec.ts [hunk 4]
```diff
@@ -473,7 +473,7 @@ describe('Coop Correction component', () => {
     // These validation flags plus more make up isCorrectionValid
     // This property is used enable/disable the file and pay button.
     expect(store.isCorrectionValid).toBe(false)
-    expect(store.hasCorrectionDataChanged).toBe(false)
+    // expect(store.hasCorrectionDataChanged).toBe(false)
```
Comment (severinbeauvais) [inline tests/unit/CoopCorrection.spec.ts:476]: Delete obsolete code, here and in other test suites below. Thanks.

**Review by severinbeauvais — APPROVED** [review]
I have tested a no-change correction for the 3 test businesses and all showed the fee summary and I was able to file them after entering just a detail comment 👍 

Also, it looks like the Filer processed the corrections 👍 

The application output still needs to be updated (separate ticket).

Pre-approved as I trust you to make the small changes above.

**Review by ArwenQin — APPROVED** [review]
Looks good!

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-edit-dev--pr-605-ksxvmv8i.web.app

**Pull request merged**